certsuite/harness.py

```diff
@@ -41,6 +41,7 @@
 import report
 
 logger = None
+remove_after_mcts = False
```

Comment:
What does mcts mean? Can you please make the variable name more meaningful?
Maybe something like "remove_marionette_after_run".